sample/README.md

```diff
@@ -7,6 +7,9 @@ The Sample describes several demos of quickly spinning up different Prebid Serve
 In the Sample, we use `docker` and `docker-compose` to instantiate examples; with docker providing a unified setup and interface,  you can spin up a demo server instance locally with only one command without knowing all the complexities.
 The docker image used in `docker-compose.yml` is the `Dockerfile` residing in the root level of the repository. 
 
+> [!NOTE] 
+> Depending on what version of Docker is installed on your system, `docker-compose` might be considered deprecated.  If you receive an error when trying to run `docker-compose`, use `docker compose` (without the dash) instead.
+
```

Comment:
Good point! 
Do you think it will give more context if we modify it to this:
```
Depending on what version of Docker is installed on your system, `docker-compose` might be considered deprecated.

Run `docker-compose` for docker compose version 1.X.
Run `docker compose` (without the dash) for docker compose versions 2.X and 3.X. 

For more details please refer to the [History and development of Docker compose](https://docs.docker.com/compose/intro/history/). 

```
I'm open to approve it as is. This is a minor suggestion.